src/translations/sqlb_nl.ts

```diff
@@ -0,0 +1,7344 @@
+    </message>
+</context>
+<context>
+    <name>ForeignKeyEditor</name>
+    <message>
+        <location filename="../ForeignKeyEditorDelegate.cpp" line="19"/>
+        <source>&amp;Reset</source>
+        <translation>&amp;Herstellen</translation>
+    </message>
+    <message>
+        <location filename="../ForeignKeyEditorDelegate.cpp" line="22"/>
+        <source>Foreign key clauses (ON UPDATE, ON DELETE etc.)</source>
+        <translation>Vreemde-sleutelclausules (ON UPDATE, ON DELETE, etc.)</translation>
+    </message>
+</context>
+<context>
+    <name>ImageViewer</name>
+    <message>
+        <location filename="../ImageViewer.ui" line="14"/>
+        <source>Image Viewer</source>
+        <translation>Afbeeldingenbekijker</translation>
+    </message>
+    <message>
+        <location filename="../ImageViewer.ui" line="92"/>
+        <source>Reset the scaling to match the original size of the image.</source>
+        <translation>Schaal de afbeelding terug naar diens oorspronkelijke grootte.</translation>
+    </message>
+    <message>
+        <location filename="../ImageViewer.ui" line="103"/>
+        <source>Set the scaling to match the size of the viewport.</source>
+        <translation>Schaal de afbeelding naar de grootte van het venster.</translation>
+    </message>
+    <message>
+        <location filename="../ImageViewer.ui" line="123"/>
+        <source>Print...</source>
+        <translation>Afdrukken...</translation>
+    </message>
+    <message>
+        <location filename="../ImageViewer.ui" line="126"/>
+        <source>Open preview dialog for printing displayed image</source>
+        <translation>Open voorvertoningsdialoogvenster om getoonde afbeelding af te drukken</translation>
+    </message>
+    <message>
+        <location filename="../ImageViewer.ui" line="129"/>
+        <source>Ctrl+P</source>
+        <translation></translation>
+    </message>
+</context>
+<context>
+    <name>ImportCsvDialog</name>
+    <message>
+        <location filename="../ImportCsvDialog.ui" line="14"/>
+        <source>Import CSV file</source>
+        <translation>CSV-bestand importeren</translation>
+    </message>
+    <message>
+        <location filename="../ImportCsvDialog.ui" line="25"/>
+        <source>Table na&amp;me</source>
+        <translation>&amp;Tabelnaam</translation>
+    </message>
+    <message>
+        <location filename="../ImportCsvDialog.ui" line="38"/>
+        <source>&amp;Column names in first line</source>
+        <translation>&amp;Kolomnamen op eerste regel</translation>
+    </message>
+    <message>
+        <location filename="../ImportCsvDialog.ui" line="55"/>
+        <source>Field &amp;separator</source>
+        <translation>&amp;Veldscheidingsteken</translation>
+    </message>
+    <message>
+        <location filename="../ImportCsvDialog.ui" line="68"/>
+        <source>,</source>
+        <translation></translation>
```

Comment:
I think it's a symbol that doesn't need translation, so I think you've left the translation blank.
I also have a similar case in the past, but there was an comments that it is safer to copy the original text for symbol
that do not require translation. Please refer to [here](https://github.com/sqlitebrowser/sqlitebrowser/pull/2329#issuecomment-658202890) for details.